src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -71,19 +71,19 @@ public Office365ConnectorWebhookNotifier(Run run, TaskListener listener) {
     }
 
     public void sendBuildStartedNotification(boolean isFromPreBuild) {
+        WebhookJobProperty property = (WebhookJobProperty) job.getProperty(WebhookJobProperty.class);
+        if (property == null || property.getWebhooks() == null || property.getWebhooks().size() == 0) {
+            log("No webhooks to notify");
```

Comment:
so then you will be printing this message for every job for every execution?

Reply:
The exiting implementation already prints the log statement for every execution, however, it does so after creating the Card object, which is a non-trivial action.

I've kept the existing implementation of logging the message in-tact, but would have no personal issue with it being removed as well.

Reply:
scratch that ... previously the message was only reported if property was null and I now capture additional conditions to avoid creating the card.

i can adjust the code to maintain the behavior of only logging when the property is null if that's desirable.  for what it's worth, the message will be displayed now when the property is null, or also when the property exists but no actual webhooks are defined.  the message of `No webhooks to notify` seems better aligned with the new implementation.

please let me know if you'd like me to update the PR to accommodate this.

Reply:
Recommend to take a look at #63 and don't repeat mistake we've already which did not make our uses happy

Reply:
I'll go ahead and remove both of the log statements then